server/graphql/vaccine_course/src/types/vaccine_course.rs

```diff
@@ -0,0 +1,70 @@
+use async_graphql::*;
+
+use graphql_core::simple_generic_errors::NodeError;
+use repository::vaccine_course::vaccine_course_row::VaccineCourseRow;
+use service::ListResult;
+
+#[derive(PartialEq, Debug)]
+pub struct VaccineCourseNode {
+    pub vaccine_course: VaccineCourseRow,
+}
+
+#[derive(SimpleObject)]
+pub struct VaccineCourseConnector {
+    total_count: u32,
+    nodes: Vec<VaccineCourseNode>,
+}
+
+#[Object]
+impl VaccineCourseNode {
+    pub async fn id(&self) -> &str {
+        &self.row().id
+    }
+
+    pub async fn name(&self) -> &str {
+        &self.row().name
+    }
+
+    pub async fn program_id(&self) -> &str {
+        &self.row().program_id
+    }
+
+    pub async fn demographic_indicator_id(&self) -> &str {
+        &self.row().demographic_indicator_id
+    }
+    // TODO Loaders for the program and demographic_indicator
```

Comment:
I wonder am I understanding right that you would also have loaders for vaccine course schedule and items tables?

Reply:
Yes you're right. I think we will.